### DISCUSSION TITLE
Disable text input into MDXEditor

### DISCUSSION THREAD

**Discussion by ghost** [mdx-editor/editor#88] [Q&A] [answered]
Hello, I really like to use this MD editor, but I have a following problem: in some cases I need to make editor read-only. I've tried something like this below, but haven't been successful. Maybe you know how to do it?
```
 <MDXEditor
      markdown={markdown}
      plugins={WRAPPED_PLUGS}
      placeholder="Comment..."
      contentEditableClassName="text-color pb-2 min-h-[44px] prose prose-brand text-sm"
      disabled
 />
```

**Comment by petyosi** [comment, marked as answer]
There's a readOnly boolean property available.

**Reply by ghost** [reply]
Thanks for your answer! That's helpful
